src/Compilers/VisualBasic/Portable/Syntax/SyntaxFacts.vb

```diff
@@ -1171,5 +1172,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Return False
         End Function
+
+        ''' <summary>
+        ''' Checks whether the element name is reserved.
+        '''
+        ''' For example:
+        ''' "Item3" is reserved (at certain positions).
```

Comment:
What does `at certain positions` refer to? It looks like `"Item3"` will always return `True`.